### PR TITLE
Improve restaurants suggestions

### PR DESCRIPTION
**Pull request by sjmiles**
Ref #2567

### CODE REVIEW

File: modalities/dom/components/elements/model-img.js [hunk 1]
```diff
@@ -11,33 +11,45 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
...
   }
   _update({src, url, fadems}, state) {
     const fade = fadems || 150;
-    const img = this.img;
-    if (img && !state.listener) {
-      state.listener = img.addEventListener('load', e => this.onLoad(img, e));
-    }
-    if (img && src) {
+    if (src && state.src !== src) {
+      //log(src);
```
Comment (piotrswigon) [inline modalities/dom/components/elements/model-img.js]: Delete log() here and below?

File: particles/Restaurants/Reservations.recipes [hunk 2]
```diff
@@ -48,12 +39,15 @@ particle ReservationMultiAnnotation in 'source/ReservationAnnotation.js'
...
 
 recipe MakeReservations
-  // &calendarReserve
   use as restaurants
-  use #reservation as event
   use #selected as restaurant
-  create as descriptions
+  create #reservation as event
+  create as calendar_descriptions
+  create as annotation_descriptions
```
Comment (piotrswigon) [inline particles/Restaurants/Reservations.recipes]: odd question: I thought we're following camelCase, I understand parser is happy with either but should we at least agree internally?

File: modalities/dom/components/elements/model-img.js (hunk 1 above)
Comment (sjmiles) [reply modalities/dom/components/elements/model-img.js]: model-img still needs work, so I left some stuff in there.

File: particles/Restaurants/Reservations.recipes (hunk 2 above)
Comment (sjmiles) [reply particles/Restaurants/Reservations.recipes]: I don't know why I did it that way, I'll change it.

**Pull request merged**